### ISSUE TITLE
RWKV : support attention_mask

### ISSUE THREAD

**Issue by pfeatherstone** [huggingface/transformers#27210]
### Feature request

At the moment, `attention_mask` in `RwkvModel` is ignored.
It would be great if this were supported.

### Motivation

I want to be able to train on a padded batch.

### Your contribution

RWKV is a bit alien to me

**Comment by pfeatherstone** [comment]
Or is it that you don't need `attention_mask` because RWKV is causal by definition? So you can just ignore padded tokens?

**Comment by ArthurZucker** [comment]
cc @younesbelkada if you remember about this! 🤗

**Comment by ArthurZucker** [comment]
RWKV does not use attention so no attention mask 😉
